cli/src/pcluster/api/util.py

```diff
@@ -11,7 +11,10 @@
 import datetime
 import logging
 import shutil
-import subprocess
+
+# A nosec comment is appended to the following line in order to disable the B404 check.
+# In this file the input of the module subprocess is trusted.
+import subprocess  # nosec
```

Comment:
Nitpick.
`# nosec B404` or even a more speaking `# nosec import_subprocess` should be safer, even if in this specific case it is pretty the same.

Reply:
Done